Makefile

```diff
@@ -217,6 +217,7 @@ SCHEMA_TEST_EXAMPLES := \
 	nmdc_example_database \
```

Comment:
Actually, we're not using this test framework anymore. We have switched to linkml-run-examples. I will try to port this test data.

Reply:
I have back-merged main into this branch and am going to keep main's Makefile but will be sure to use your new test data